src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/InsertHelpers.kt

```diff
@@ -190,12 +191,6 @@ fun giveUserGroupPermission(groupName: String,
 
         if (addPermission)
         {
-            it.dsl.insertInto(ORDERLYWEB_PERMISSION)
```

Comment:
Not quite sure what the addPermission is meant to be doing here - it used to mean 'add to the permission table if it's not already there' but that's now removed, and it will just add the (abstract) permission if it's not there, before going on to insert the concrete permission in all cases? I would have thought potentially adding to the permission table would still be useful, and that you'd always want to to add the abstract permission if it wasn't already there...?

Reply:
Yes, this was just for the case in which a test adds the same permission scoped in different ways, I didn't want to add multiple duplicate abstract permissions. But it doesn't really matter tbh as it's only test logic and gets cleaned up, so we could just do without this and allow duplicate abstract permissions, or indeed just check if the abstract permission exists first. Maybe the latter would be clearer, will change. Re adding permissions, I'm inclined not to do that, so we can be sure we're only ever referring to `real` permissions that will exist in the db in production.